src/index.js

```diff
@@ -28,27 +60,38 @@ class UglifyJsPlugin {
 		compiler.plugin("compilation", (compilation) => {
 			if(options.sourceMap) {
 				compilation.plugin("build-module", (module) => {
-					// to get detailed location info about errors
+          // to get detailed location info about errors
 					module.useSourceMap = true;
 				});
 			}
 			compilation.plugin("optimize-chunk-assets", (chunks, callback) => {
-				const files = [];
-				chunks.forEach((chunk) => files.push.apply(files, chunk.files));
-				files.push.apply(files, compilation.additionalChunkAssets);
-				const filteredFiles = files.filter(ModuleFilenameHelpers.matchObject.bind(undefined, options));
-				filteredFiles.forEach((file) => {
-					const oldWarnFunction = uglify.AST_Node.warn_function;
-					const warnings = [];
-					let sourceMap;
+				const files = compilation.additionalChunkAssets;
+				let i = chunks.length - 1;
+				while(i--) {
+          // eslint-disable-next-line prefer-spread
```

Comment:
bad indent here too

looks like the comments were indented with spaces while the rest of the file is indented with tabs?